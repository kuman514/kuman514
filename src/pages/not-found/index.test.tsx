import { render } from '@testing-library/react';

import NotFoundPage from '.';

describe('NotFoundPage', () => {
  it('should show as expected', () => {
    const { container } = render(<NotFoundPage />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
