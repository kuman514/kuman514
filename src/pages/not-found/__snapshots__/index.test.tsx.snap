// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage should show as expected 1`] = `
<main
  class="flex min-h-screen flex-col justify-center items-center gap-4 text-center"
>
  <h1
    class="text-bold text-4xl sm:text-5xl md:text-6xl"
  >
    404 Not Found
  </h1>
  <span>
    요청하신 페이지를 찾을 수 없습니다.
  </span>
</main>
`;
